src/Runner.Common/RunnerDotcomServer.cs

```diff
@@ -163,6 +164,8 @@ private async Task<DistributedTask.WebApi.Runner> AddOrReplaceRunner(int runnerG
                         {"ephemeral", agent.Ephemeral},
                         {"labels", agent.Labels},
                         {"public_key", publicKey},
+                        {"os", VarUtil.OS},
```

Comment:
This or the combined package name, ex Linux-x64

Reply:
We send back just the `os` in the REST API, so it's easy enough to concatenate them if we need to. https://docs.github.com/en/rest/actions/self-hosted-runners?apiVersion=2022-11-28